template/lambda-sqs-worker-cdk/infra/index.ts

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env node
```

Comment:
Does the shebang do anything given this is TypeScript source code and there's a script to run it through `ts-node`?

Reply:
It's redundant, so I've removed it.